package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "fusion-plugin-redux-action-emitter-enhancer",
   "description": "Redux store enhancer that emits actions via an injected event emitter",
-  "version": "0.2.0",
+  "version": "0.2.1",
```

Comment:
Technically breaking, since we now destructure `deps <-> deps.emitter`

Reply:
Ah yeah, but I think it makes sense to ship under 0.2.x since it's broken 😅